.gitignore

```diff
@@ -3,3 +3,5 @@ env/*
 
 sms_tracking/config.py
 .env
+sms_tracking/.DS_Store
+/.DS_Store
```

Comment:
You can just do `.DS_Store` and it will ignore in all directories. https://git-scm.com/docs/gitignore